sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/models/CertificateOperation.java

```diff
@@ -110,7 +110,7 @@ public String getCertificateType() {
      *
      * @return the certificateTransparency status.
      */
-    public Boolean getCertificateTransparency() {
+    public boolean getCertificateTransparency() {
```

Comment:
should this be `isCertificateTransparent` ?

Reply:
yup, fixed